Library/Formula/bazaar.rb

```diff
@@ -27,5 +28,23 @@ def install
     libexec.install 'bzr', 'bzrlib'
 
     bin.install_symlink libexec+'bzr'
+
+    # install the plugins under /var/bazaar/plugins/
+    plugins_orig = libexec/"bzrlib/plugins"
+    plugins_new = var/"bazaar/plugins"
+
+    Dir[plugins_orig/"*"].each do |plugin|
+      path = Pathname.new plugin
+      plugins_new.install plugin unless File.exist? (plugins_new/path.basename)
+    end
```

Comment:
@mikemcquaid Will it work if `var/bazaar/plugins` already exists and already have some plugins in it?